### PR TITLE
ToastAndNavigation

### PR DESCRIPTION
**Pull request by MayankKumar10**
Adding Toast And Navigation into component Library

### CODE REVIEW

File: src/styles/Aside.css [hunk 1]
```diff
@@ -13,7 +13,7 @@
   justify-content: space-around;
   cursor: pointer;
   color: #fff;
```
Comment (ShekharDhangar) [inline src/styles/Aside.css:15]: try to use variables for color

File: src/styles/Aside.css [hunk 2]
```diff
@@ -13,7 +13,7 @@
   justify-content: space-around;
   cursor: pointer;
```
Comment (ShekharDhangar) [inline src/styles/Aside.css:14]: also for such generic properties, use helper class

File: src/styles/Modal.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
```
Comment (ShekharDhangar) [inline src/styles/Modal.css:3]: use helper class as fixed

File: src/styles/Modal.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 1.5rem;
+  border: 1px solid #888;
+  width: 80%;
+}
+
+.close {
+  color: #aaaaaa;
```
Comment (ShekharDhangar) [inline src/styles/Modal.css:24]: use color variables

File: src/styles/Navigation.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
+.AppBarContainer {
+  display: flex;
+  width: 25em;
+  height: 3em;
+  align-items: center;
```
Comment (ShekharDhangar) [inline src/styles/Navigation.css:5]: use helper class flex

**Review by ShekharDhangar — APPROVED** [review]
Please do the changes, rest is good

File: src/styles/Aside.css (hunk 2 above)
Comment (MayankKumar10) [reply src/styles/Aside.css:14]: Okay, I'll do that

File: src/styles/Modal.css (hunk 4 above)
Comment (MayankKumar10) [reply src/styles/Modal.css:24]: Okay, I'll do that

File: src/styles/Navigation.css (hunk 5 above)
Comment (MayankKumar10) [reply src/styles/Navigation.css:5]: Okay, I'll do that

File: src/components/Badge.html [hunk 6]
```diff
@@ -57,33 +57,33 @@ <h3 class="h3Tag">Basic Badge And Color Badge</h3>
...
     <h3 class="h3Tag">With Badge</h3>
-    <p class="descriptionOne">Badge create Status of Badge</p>
+    <p class="descriptionOne discriptionText">Badge create Status of Badge</p>
 
     <div class="DOMContainer flex-row-center">
     
     <div class="AvatarDomMainContainer">
-        <div class="AvatarDomContainer descriptionOne statusAvatarFlexRightTop">
+        <div class="descriptionOne statusBadgeFlexRightTop">
         <img class="AvatarImage smallAvatar box-shadow-round" src="../Images/steve.jpg" alt="" sizes="" srcset="">
```
Comment (Dev-AkshatGupta) [inline src/components/Badge.html:66]: Please remove the undesired attributes form img and also please add alt to the image tag

File: src/components/Alert.html [hunk 7]
```diff
@@ -14,12 +14,12 @@
 
   <div class="AlertContainerOne">
```
Comment (Dev-AkshatGupta) [inline src/components/Alert.html:15]: Mayank Improve the Indentation

Comment (MayankKumar10) [reply src/components/Alert.html:15]: Okay, I'll do that. Thank you Akshat

**Pull request merged**